toolchain/mfc/run/run.py

```diff
@@ -102,7 +102,7 @@ def __generate_job_script(targets, case: input.MFCInputFile):
         case=case,
         MFC_ROOTDIR=MFC_ROOTDIR,
         qsystem=queues.get_system(),
-        profiler=__profiler_prepend(),
+        profiler=' '.join([ shlex.quote(_) for _ in __profiler_prepend()])
```

Comment:
Actually I think `shlex.join(__profiler_prepend())` would do the trick :). I just learned about its existence.